components/automate-gateway/api/iam/v2/common/tokens.proto

```diff
@@ -4,11 +4,19 @@ package chef.automate.api.iam.v2;
 option go_package = "github.com/chef/automate/components/automate-gateway/api/iam/v2/common";
 
 message Token {
+    // Unique, optionally user-specified ID. Cannot be changed.
     string id = 1;
+    // User-specified name.
     string name = 2;
+    // Unique, optionally user-specified value.
     string value = 3;
+    // Active state. Defaults to true.
+    // If set to false, token will not authenticate.
     bool   active = 4;
+    // Created timestamp.
     string created_at = 5;
+    // Updated timestamp.
     string updated_at = 6;
+    // Array of projects token is in. Empty by default.
```

Comment:
Again we should strive for one voice, one way or the other.
Policy endpoints use "The list of projects this policy belongs to." But I like your "Empty by default", too.

(Same consistency comments on the other tokens.proto file too)